src/components/elements/annotationSidePanel/components/annotationWizard/steps/NewInstance.tsx

```diff
@@ -0,0 +1,166 @@
+/* Import Dependencies */
+import classNames from 'classnames';
+import jp from 'jsonpath';
+import { useState } from 'react';
+import { Row, Col, Card } from 'react-bootstrap';
+
+/* Import Utilities */
+import { MakeReadableString } from 'app/Utilities';
+import { MakeJsonPathReadableString } from 'app/utilities/SchemaUtilities';
+
+/* Import Hooks */
+import { useTrigger } from 'app/Hooks';
+
+/* Import Types */
+import { DigitalSpecimen } from 'app/types/DigitalSpecimen';
+import { DigitalMedia } from 'app/types/DigitalMedia';
+import { AnnotationTarget, Dict, ParentClass } from 'app/Types';
+
+/* Import Components */
+import ParentClassification from './ParentClassification';
+import { Button } from 'components/elements/customUI/CustomUI';
+
+
+/* Props Type */
+type Props = {
+    superClass: DigitalSpecimen | DigitalMedia | Dict,
+    selected: boolean,
+    annotationTarget?: AnnotationTarget,
+    formValues?: Dict,
+    SetFieldValue?: Function
+};
+
+
+/**
+ * Component that renders a new, annotatable instance of the selected annotation target in the select instance step of the wizard
+ * @param superClass The provided super class
+ * @param selected Boolean indicating if the new instance option has been selected
+ * @param annotationTarget The annotation target set
+ * @param formValues The current values in the annotation wizard form
+ * @param SetFieldValue Function to set a value of a field in the form
+ * @returns JSX Component
+ */
+const NewInstance = (props: Props) => {
+    const { superClass, selected, annotationTarget, formValues, SetFieldValue } = props;
+
+    /* Hooks */
+    const trigger = useTrigger();
+
+    /* Base variables */
+    const [parentClasses, setParentClasses] = useState<ParentClass[]>([]);
+
+    /* Check for parent classes, they either contain the term 'has' or start with a capital */
+    trigger.SetTrigger(() => {
+        const parentClasses: ParentClass[] = [];
```

Comment:
maybe move this to a helper function for readability

Reply:
Done